.42AI/init.sh

```diff
@@ -11,8 +11,8 @@ COLOR_PURPLE="\e[1;35m"
 COLOR_CYAN="\e[1;36m"
 COLOR_RESET="\e[0m"
 
-PATH_PYTHON=`where python3`
-PATH_ENVRC=`pwd`/.envrc
+PATH_PYTHON=`type -a python3`
```

Comment:
What is the difference between `where python3` and `type -a python3` ?
When I run those commands I got:
```sh
➜  ~ type -a python3
python3 is /usr/bin/python3
python3 is /bin/python3
➜  ~ where python3
/usr/bin/python3
/bin/python3
```
Does PATH_PYTHON env variable will still be correct ?

Reply:
`where` is not a bash command so the script raises an error at the beginning.
The equivalent bash command is `type -a`